deploy/runtools/run_farm_deploy_managers.py

```diff
@@ -148,6 +148,11 @@ def disconnect_all_nbds_instance(self) -> None:
 
                 run("; ".join(fullcmd))
 
+    def get_remote_sim_dir_for_slot(self, slotno: int) -> str:
```

Comment:
docstring

Reply:
I am taring things locally. If I may ask, which part led you to think that the tar was being done remotely ?

Reply:
I didn't read your in-source documentation correctly. 

I think it was the confluence of seeing:
1) `run` commands, which generally imply running on a worker node (as opposed to `local`) and forgetting that we've switched to using `run` + localhost in favor of `local` generally.
2) See rsync + "remote_dir" in places where the remote dir is just another dir under output. Again, if i read the code more carefully that would've been clear.